hack/release-scripts/generate-release-pr

```diff
@@ -88,10 +88,14 @@ parse_args() {
 
 update_readme() {
   log "Updating README.md"
-  # vi macro that adds new driver version 'Container Images' row to README.md
-  vi -s <(echo "gg/## Container Images
-  jjjjyy:%s/${PREV_DRIVER_VERSION}/${NEW_DRIVER_VERSION}/g
-  jjjjjjp:wq") "$README_PATH"
+  $SED "/^## Container Images$/,+5 s/$PREV_DRIVER_VERSION/$NEW_DRIVER_VERSION/" -i "$README_PATH"
+
+  # If the major/minor version has changed, we also need to replace the old prev version with the new previous version
+  # (If not, that means this is a patch release, and we shouldn't change the other version)
+  if [ "${PREV_DRIVER_VERSION%.*}" != "${NEW_DRIVER_VERSION%.*}" ]; then
```

Comment:
Thanks for updating this script as well. Glad we no longer have to rely on a vi macro.